src/locales/translations/de.ts

```diff
@@ -0,0 +1,79 @@
+  nickname: "Nickname",
+  settings: "Eintstellungen",
+  minimize: "Minimieren",
+  maximize: "Maximieren",
+  close: "Schließen",
+  add_server_modal_description_1: "Server manuell zu Favoriten hinzufügen.",
+  add_server_modal_description_2: "Beispiel: 127.0.0.1:7777",
+  add: "Hinzufügen",
+  server: "Server",
+  address: "Adresse",
+  players: "Spieler",
+  server_join_prompt_enter_password:
+    "Dieser Server ist passwortgeschützt, bitte gib ein Passwort ein.",
+  server_join_prompt_enter_password_input_placeholder: "Passwort eingeben...",
+  server_join_prompt_nickname_input_placeholder: "Nickname eingeben...",
+  connect: "Verbinden",
+  copy: "Kopieren",
+  remove_from_favorites: "Aus Favoriten entfernen",
+  add_to_favorites: "Zu Favoriten hinzufügen",
+  filters: "Filter",
+  filter_only_omp_servers: "Nur open.mp Server",
+  filter_non_empty_servers: "Server mit Spielern",
+  filter_unpassworded_servers: "Server die nicht passwortgeschützt sind",
+  rule: "Regel",
+  value: "Wert",
+  player: "Spieler",
+  score: "Punkte",
+  locked: "Gesperrt",
+  unlocked: "Entsperrt",
+  openmp_server: "open.mp Server",
+  name: "Name",
+  ping: "Ping",
+  mode: "Modus",
+  filter_servers: "Server filtern",
+  search_for_server_hostname_mode: "Anhand von Hostname oder Modus nach Server suchen",
+  clear_recently_joined_list: "Liste kürzlich betretener Server leeren",
+  refresh_servers: "Server Liste aktualisieren",
+  play: "Spielen",
+  remove_selected_server_from_favorites:
+    "Ausgewählten Server aus Favoriten entfernen",
+  add_selected_server_to_favorites: "Ausgewählhten Server zu Favoriten hinzufügen",
+  add_server: "Server hinzufügen",
+  hide_player_and_rule_list: "Spielerliste und Regeln verstecken",
+  show_player_and_rule_list: "Spielerliste und Regeln anzeigen",
+  copy_server_info: "Server Info kopieren",
+  settings_gta_path_input_label:
+    "GTA: San Andreas PFad (Ort der SA:MP Installation)",
+  browse: "Durchsuchen",
+  settings_import_nickname_gta_path_from_samp:
+    "Nickname and gtasa Pfad aus SA-MP Einstellungen übernehmen",
+  settings_import_samp_favorite_list: "Favoruiten aus SA-MP importieren",
+  settings_reset_application_data:
+    "Alle Daten auf Werkseinstellungen zurücksetzungen (Löscht Einstellungen und alle Serverlisten)",
+  settings_new_update_available: "⚠  Aktualisierung verfügbar. Zum Downloaden, klicken! ⚠",
+  settings_credits_made_by: "Mit ❤️ gemacht, von",
+  settings_credits_view_source_on_github: "Quellcode auf GitHub ansehen",
+  update_modal_update_available_title: "Aktualisierung verfügbar!",
+  update_modal_update_available_description:
+    'Neu Launcher-Version verfügbar!\nDeine Launcher Build-Version: {{ version }}\nAktuellste Launcher Build-Version: {{ newVersion }}\nKlicke "Download" um die Seite der veröffentlichten Launcher zu öffnen.',
+  download: "Download",
+  update_modal_remind_me_next_time: "Erinnere mich später",
+  update_modal_skip_this_update: "Aktualisierung auslassen",
```

Comment:
Again, I wouldn't translate Update.

```suggestion
  update_modal_skip_this_update: "Update überspringen",
```

Reply:
Right, but Anglizismen are often used, just like everyone says "downloaden" instead of "herunterladen". We should at least keep everything consistent and adjust "downloaden" here aswell: https://github.com/openmultiplayer/launcher/pull/15#discussion_r1390414622